dataset/download.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+wget https://zenodo.org/record/1257463/files/pria_rmi_cv.tar.gz
+wget https://zenodo.org/record/1257463/files/pria_rmi_pcba_cv.tar.gz
+wget https://zenodo.org/record/1257463/files/pria_prospective.csv.gz
```

Comment:
Should we move this file as well?  Where is it expected to be?

Reply:
@agitter I put them in the default (current) folder so as to make sure we are following same file structure.